packages/gasket-plugin-nextjs/docs/deployment.md

```diff
@@ -128,3 +128,20 @@ Follow the [Docker deployment guide] to see a sample `Dockerfile`.
 
 [config]: /packages/gasket-cli/docs/configuration.md
 [Docker deployment guide]: docker-deployment.md
+
+## Gotchas
+
+### Cache directory
+
+The Gasket CLI is built upon `@oclif` and uses some plugins that need access to
+read/write to a cache directory. Based on the [oclif docs], this is configured
+to the following defaults:
+ - macOS: `~/Library/Caches/@gasket/cli`
+ - Unix: `~/.cache/@gasket/cli`
+ - Windows: `%LOCALAPPDATA%\@gasket\cli`
+
+For some deployment environments, this may need to be adjusted from the
+defaults. To override where the cache directory is for your deployment, you can
+set the `GASKET_CACHE_DIR` env variable, such as in the `Dockerfile`.
```

Comment:
Is that just the cache directory for oclif?

I like the idea of being able to set a _base_ cache directory for all things Gasket, and then specific cache directories would fall under that, like `$GASKET_CACHE_DIR/oclif`. That way we won't have the need in the future for multiple environment variables.

Reply:
Correct, it's the cache dir for oclif, and it is generated based on the cli name (`gasket`). It is not something we control directly. However, it seems like Gasket plugins and such could utilize it as a base dir for caching also.

Reply:
OK, just want to make sure it _can_ be used as a general purpose base cache directory, that oclif wouldn't cause conflicts with other caches; otherwise maybe it could be an oclif subdirectory under that base cache directory.

Reply:
👍 I'll make a ticket to investigate using it for this aspect more thoroughly and we can provide some docs for plugin authors on it.